packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -24,8 +27,20 @@ export interface ISummaryStats {
     unreferencedBlobSize: number;
 }
 
+/**
+ *  The summarization methods are recursively invoked during the summary calculation and will
+ *  compose the Summary Tree. At the same time, each component that is taking part of the summarization
+ *  will populate the tree information aggregation for its subtree through the ISummaryStats interface.
+ *  Any component that implements IChannelContext, IFluidDataStoreChannel or extends SharedObject
+ *  will be taking part of the summarization process.
+ */
 export interface ISummaryTreeWithStats {
+    /** Represents an agreggation of node counts and blob sizes associated to the current summary information */
     stats: ISummaryStats;
+    /**
+     * Summary defines a recursive data structure that will be converted to a snapshot tree and uploaded
+     * to the backend.
```

Comment:
We can be more direct with the language, and thus less verbose.
Phrasing using "defines" and "represents" are common examples of this, and should be avoided unless we explicitly need to distinguish between some real phusical thing the code is describing and the model the types in the code create. Here this field IS the summary tree. There is no reason to put "Represents" on every type and "defines" on every field.
For example here:
```suggestion
     * A recursive data structure that will be converted to a snapshot tree and uploaded
     * to the backend.
```

Also in this case we need to consider what documenation belongs on the type, and what belongs on the field. We don't want to be redundent and include docs that belong on the type everywhere its used.

Whats relevent here is that this is the tree that is described by the stats.

One other think to look out for: documentation should be clear about what a type can be used for. Is ISummaryTreeWithStats only use before a summary is uploaded (and not held onto after, or to represent downloaded summaries)? Is so, the documentation on this type chould be clear about that. If not, the docs on this field are too specific.

As far as I can tell, the purpose of this type (ISummaryTreeWithStats) is to pair a summary with some stats about it, so I think just documenting this field as "the summary decribed by stats" would be fine: leave detaisl about how ISummaryTree is used for documenation on that type, or the thing that uses it.